W-13789613 pdk prereqs gr (#355)
mulesoft/docs-gateway: gateway/1.5/modules/ROOT/pages/policies-pdk-prerequisites.adoc

> = Reviewing Policy Development Kit Prequisites

Reviewer: As far as I can tell, there is no such word as Prequisites. It is Prerequisites. I see that you changed it her and in the file name in nav.adoc.

** https://gnuwin32.sourceforge.net/packages/make.htm[Make for Windows ^]
** Chocolatey: 
+
[source,ssh]
----
choco install make
----

[[cli]]
== Anypoint CLI 

To configure the Anypoint CLI for PDK:

. xref:anypoint-cli::install.adoc[Install Anypoint CLI 4.x].
. xref:anypoint-cli::auth.doc[Authenticate to the Anypoint Platform CLI] using a xref:access-management::connected-apps-overview.adoc[Connected App].
+
Authenticate with a Connected App by providing the app's client ID and client secret.
+
NOTE: No extra permissions are required to download the Anypoint CLI PDK Plugin. However, to complete xref:policies-pdk-publish-policies.adoc[], the Connected App must have the `Exchange Contributor` scope applied.

. Run the following command to install the Anypoint CLI PDK Plugin:
+
[source,ssh]
----
anypoint-cli-v4 plugins:install @mulesoft/anypoint-cli-pdk-plugin
----

== Docker 

Docker is only required to use the xref:policies-pdk-debug-local.adoc[local testing environment] provided with PDK. You can use all other PDK features without docker.

To install Docker, see https://docs.docker.com/engine/install/[Install Docker Engine ^].

== Visual Studio (Suggested IDE)

> You can use PDK in any IDE or text editor. However, MuleSoft suggests that you use https://code.visualstudio.com/[Microsoft Visual Studio ^] with the https://code.visualstudio.com/docs/languages/rust[Rust in Visual Studio Code Plugin ^] installed.

Reviewer: ```suggestion
You can use PDK in any IDE or text editor, such as https://code.visualstudio.com/[Microsoft Visual Studio ^] with the https://code.visualstudio.com/docs/languages/rust[Rust in Visual Studio Code Plugin ^] installed.
```

Reviewer: Please see the [Style Guide, MuleSoft Recommends section](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference). We are supposed to avoid "MuleSoft recommends" or MuleSoft suggests". Is there any reason why we are recommending the editor here? If so, you should add why this is the one we are suggesting. Here I just changed it to show it as an example, which is the same as suggesting it.

Author: Yeah, that makes sense. It is only really suggested because it provides the Rust plugin and there will be a mulesoft PDK VScode plugin for GA, but I guess we can't mention that in the docs yet. I think your suggestion works for now.
